vertx-sql-client/src/main/java/io/vertx/sqlclient/impl/Utils.java

```diff
@@ -0,0 +1,42 @@
+package io.vertx.sqlclient.impl;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+import static io.vertx.sqlclient.Tuple.JSON_NULL;
+
+/**
+ * Bunch of generic utils.
+ */
+public final class Utils {
```

Comment:
You might as well use an interface and a static method. I know we old Java-ers don't think of interfaces for static methods in the first place, but you can actually do it 😸

Reply:
I don't have any preferences, some use a Java enum as well

I think the Java language is missing the keyword _singletion_ :-)